changelogs/fragments/allow_listing_known_inhibitors.yml

```diff
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - Move collection dependencies from galaxy.yml to requirements.yml
```

Comment:
Remove 2 lines above. Cut and paste error? :scissors:

Reply:
D'oh.. Fixed now!